apps/withdraw-pool/src/context/iframe.tsx

```diff
@@ -66,6 +92,12 @@ export function IFrameContextProvider({ children }: PropsWithChildren) {
         publicClient,
         cowShedProxy,
         userPoolSwr,
+        cowShed,
+        hookInfo,
+        setHookInfo,
+        signer,
+        actions,
+        jsonRpcProvider,
       }}
     >
       {children}
```

Comment:
It is not about the current PR changes, actually, but wouldn't be better to pass this userPoolSwr to another context? Then we could let this IFrame context more generalized for all hooks